backend/src/v5/models/tickets.comments.js

```diff
@@ -47,8 +47,17 @@ TicketComments.getCommentById = async (teamspace, project, model, ticket, _id,
 };
 
 TicketComments.getCommentsByTicket = (teamspace, project, model, ticket,
-	projection = { teamspace: 0, project: 0, model: 0, ticket: 0 },
-	sort = { createdAt: -1 }) => findMany(teamspace, { teamspace, project, model, ticket }, projection, sort);
+	{
+		projection = { teamspace: 0, project: 0, model: 0, ticket: 0 },
+		updatedSince,
+		sort = { createdAt: -1 },
+	} = {}) => {
+	const query = { teamspace, project, model, ticket };
+	if (updatedSince) {
+		query.updatedAt = { $gt: updatedSince };
```

Comment:
The english language is ambiguous on the description of since https://english.stackexchange.com/questions/261767/since-2009-is-it-inclusive-or-exclusive
https://ell.stackexchange.com/questions/114760/its-been-x-days-since-i-last-saw-you-should-i-use-an-inclusive-or-exclusive-c

Considering this is a epoch timestamp, we're talking about a ms precision here so I think it's not too important